src/main/scala/uk/gov/nationalarchives/tdr/api/model/consignment/ConsignmentReference.scala

```diff
@@ -0,0 +1,21 @@
+package uk.gov.nationalarchives.tdr.api.model.consignment
+
+import java.io.IOException
+
+import uk.gov.nationalarchives.oci.Alphabet.Alphabet
+import uk.gov.nationalarchives.oci.{Alphabet, BaseCoder, IncludedAlphabet}
+
+object ConsignmentReference {
+  val alphabet: Either[IOException, Alphabet] = Alphabet.loadAlphabet(Right(IncludedAlphabet.GCRb25))
```

Comment:
@suzannehamilton I'm loading in the alphabet outside the `createConsignmentReference` method here, I think because I've rebased a few times things look a bit jumbled up now. Apologies if it's hard to read! :sweat_smile:

Reply:
Sorry, I didn't explain that very clearly. 😅 The error handling is still happening on every method call, but it's exactly the same every time.

If you move the error handling out of the method, `alphabet` has a much clearer type (`Alphabet`, rather than `Either[IOException, Alphabet]`), and the error handling only happens once. It also has the useful effect that the error handling will happen when the app starts up. This means we'll catch the error on deployment, rather than only when a consignment is created.

Reply:
Ah I see! I'll change it! :+1: